Testing/functional/tests/cypress/integration/ui/pages/vaccineCard.js

```diff
@@ -0,0 +1,545 @@
+const { AuthMethod, localDevUri } = require("../../../support/constants");
+
+const monthNames = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+];
+
+const dobYearSelector =
+    "[data-testid=dateOfBirthInput] [data-testid=formSelectYear]";
+const dobMonthSelector =
+    "[data-testid=dateOfBirthInput] [data-testid=formSelectMonth]";
+const dobDaySelector =
+    "[data-testid=dateOfBirthInput] [data-testid=formSelectDay]";
+const dovYearSelector =
+    "[data-testid=dateOfVaccineInput] [data-testid=formSelectYear]";
+const dovMonthSelector =
+    "[data-testid=dateOfVaccineInput] [data-testid=formSelectMonth]";
+const dovDaySelector =
+    "[data-testid=dateOfVaccineInput] [data-testid=formSelectDay]";
+
+const feedbackPhnMustBeValidSelector = "[data-testid=feedbackPhnMustBeValid]";
+const feedbackPhnIsRequiredSelector = "[data-testid=feedbackPhnIsRequired]";
+const feedbackDobIsRequiredSelector = "[data-testid=feedbackDobIsRequired]";
+const feedbackDovIsRequiredSelector = "[data-testid=feedbackDovIsRequired]";
+
+const homeUrl = "/";
+const vaccineCardUrl = "/vaccinecard";
+
+const vaccinationStatusModule = "VaccinationStatus";
+
+function enterVaccineCardPHN(phn) {
+    cy.get("[data-testid=phnInput]")
+        .should("be.visible", "be.enabled")
+        .type(phn);
+}
+
+function select(selector, value) {
+    cy.get(selector).should("be.visible", "be.enabled").select(value);
+}
+
+function selectExist(selector, value) {
+    cy.get(selector)
+        .children("[value=" + value + "]")
+        .should("exist");
+}
+
+function selectNotExist(selector, value) {
+    cy.get(selector)
+        .children("[value=" + value + "]")
+        .should("not.exist");
+}
+
+function clickVaccineCardEnterButton() {
+    cy.get("[data-testid=btnEnter]").should("be.enabled", "be.visible").click();
+}
+
+describe("Vaccine Card Page", () => {
+    it("Landing Page - Vaccine Card Button does not exist - Vaccine Status module disabled", () => {
+        cy.enableModules({});
```

Comment:
If you don't want to enable any modules, I'd suggest using `cy.enableModules([])`, since our implementation expects either an array of strings or a single string.

Reply:
Changed.